WordPress/Tests/Arrays/ArrayDeclarationSpacingUnitTest.inc

```diff
@@ -2,7 +2,23 @@
 
 $good = array( 'value1', 'value2' ); // Ok.
 
-$query_vars = array('post_type' => 'food'); // Bad, no spaces after opening and before closing parenthesis.
+$query_vars = array('food'); // Bad, no spaces after opening and before closing parenthesis.
 
 // Test for fixing of extra whitespace.
 $test = array(   1, 2     );
+
+$bad = array( 'key' => 'value' ); // Bad, each value of an associative array should start on a new line.
```

Comment:
I have always thought that when there are *multiple* values in the array, it should be split to have one value per line. When there is just one key => value pair, I don't think that it has to be multiline (although that should still be permissible, in case of long values that need to be wrapped). I do see that the docs actually say:

>For associative arrays, values should start on a new line.

Though I'm a bit incredulous that that means for arrays with a single element.